truss/truss_handle.py

```diff
@@ -10,15 +10,20 @@
 from typing import Callable, Dict, List, Optional, Tuple, Union
 from urllib.error import HTTPError
 
+import numpy as np
```

Comment:
The delayed import of numpy is intentional, we don't want numpy dependency for the context builder image to keep the size low. That reminds me that we need to add a test for the context builder image to catch such issues. Let me create an issue for that.

Same for python_on_whales.

Reply:
I think I need a bit more explanation to understand  this use case. Lets earmark for the in-person discussion.